src/main/java/com/controller/MailController.java

```diff
@@ -0,0 +1,37 @@
+package com.controller;
+
+import com.dto.MailDto;
+import com.model.Mail;
+import com.service.MailService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class MailController {
+
+    MailService mailService;
+
+    @GetMapping("/mails")
+    public List<Mail> getMails(){
+        return mailService.getAll();
+    }
+
+    @PostMapping()
```

Comment:
This mapping will not be under `/mails`. In order to achieve that, you have to move common part to the class level annotation. If there are no arguments passed into this annotation, then paretheses are redundant